shared/productTypes.ts

```diff
@@ -445,6 +447,46 @@ export const PRODUCT_TYPES: { [productKey in ProductTypeKeys]: ProductType } = {
 			productFilenamePart: 'Newspaper - Home Delivery',
 		},
 	},
+	nationaldelivery: {
+		productTitle: calculateProductTitle('Newspaper Delivery'),
+		friendlyName: () => 'newspaper home delivery subscription',
+		shortFriendlyName: 'newspaper home delivery',
+		productType: 'nationaldelivery',
+		groupedProductType: 'subscriptions',
+		allProductsProductTypeFilterString: 'HomeDelivery',
+		urlPart: 'nationaldelivery',
+		getOphanProductType: () => 'PRINT_SUBSCRIPTION',
+		productPageNewsletterIDs: [FRONT_PAGE_NEWSLETTER_ID],
+		softOptInIDs: [
+			SoftOptInIDs.SupportOnboarding,
+			SoftOptInIDs.SubscriberPreview,
+			SoftOptInIDs.SimilarProducts,
+			SoftOptInIDs.SupporterNewsletter,
+		],
+		holidayStops: {
+			issueKeyword: 'paper',
+		},
+		delivery: {
+			showAddress: showDeliveryAddressCheck,
+			enableDeliveryInstructionsUpdate: true,
+			records: {
+				productNameForProblemReport: 'National Delivery',
```

Comment:
same for this - does this need to match something on an api somewhere?

Reply:
This value maps to a value in the delivery problem creation in support-service-lambdas here: https://github.com/guardian/support-service-lambdas/blob/b0194326780271679c71a63096cf4c863ae1aa52/handlers/delivery-records-api/src/main/scala/com/gu/delivery_records_api/DeliveryRecordsService.scala#L55

I'm not completely sure what this value should be yet because the accompanying work hasn't been done, but I've discussed with John D who is doing that work, and will go with this for now and update later if necessary.